src/jaxsim/api/data.py

```diff
@@ -39,7 +39,7 @@ class JaxSimModelData(common.ModelDataWithVelocityRepresentation):
     contacts_params: jaxsim.rbda.ContactsParams = dataclasses.field(repr=False)
 
     time_ns: jtp.Int = dataclasses.field(
-        default_factory=lambda: jnp.array(0, dtype=jnp.uint64)
+        default_factory=lambda: jnp.array(0, dtype=int)
```

Comment:
I guess that also here we need to select either `uint64` or `uint32` depending on the JAX configuration.

Reply:
Yes, you're right. Thanks a lot! Addressed in https://github.com/ami-iit/jaxsim/pull/225/commits/c58bc190e3337f4cbb74f0b8f863612daeb74a35